ocrolib/lstm.py

```diff
@@ -645,6 +663,8 @@ def states(self):
     def weights(self):
         for w,dw,n in self.net.weights():
             yield w,dw,"Reversed/%s"%n
+    def resizeoutput(self, no, deleted_positions):
+        self.net.resizeOutput(no, deleted_positions)
```

Comment:
Python is case sensitive 'resizeOutput' != resizeoutput

Use the standard Python style for naming functions/methods
https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables